### PR TITLE
w/wawa.md

### PR DESCRIPTION
**Pull request by Betsimisaraka**
Hey team

Please have a look at my PR, happy to see the changes if need to be.

Thanks in advance
Anita

### CODE REVIEW

File: w/wawa.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Wawa
+
+![wawa](wawa.JPG)
+
+   This artist  is among the most popular contemporary salegy artists and has recorded songs in collaboration with numerous other Malagasy artists. In 2011 he released an album of traditional salegy, featuring kabosy, marovany and traditional percussion accompaniment.[2] In 2010 the band completed extensive tours to sold out audiences in France and Madagascar.[3] A music reviewer for Midi Madagasikara described Wawa in 2013 as the "perfect entertainer, who never ceases to perform at the highest levels.
+
+**Genre:** Salegy
+- Salegy is  a genre of music from the northern coastal region of Madagascar.
```
Comment (ndorfin) [inline w/wawa.md]: No need to include the genre description here, we'll have that in `/genres.md`

File: w/wawa.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Wawa
+
+![wawa](wawa.JPG)
+
+   This artist  is among the most popular contemporary salegy artists and has recorded songs in collaboration with numerous other Malagasy artists. In 2011 he released an album of traditional salegy, featuring kabosy, marovany and traditional percussion accompaniment.[2] In 2010 the band completed extensive tours to sold out audiences in France and Madagascar.[3] A music reviewer for Midi Madagasikara described Wawa in 2013 as the "perfect entertainer, who never ceases to perform at the highest levels.
```
Comment (ndorfin) [inline w/wawa.md]: Please remove the spaces at the beginning of this paragraph.
Please also remove the `[1]` etc from the bio.

**Review by ndorfin — CHANGES_REQUESTED** [review]
Another great PR, just a few small fixes please.

**Review by ndorfin — APPROVED** [review]
Wow, this is great, good work Anita! 🥇

File: w/wawa.md (hunk 2 above)
Comment (Betsimisaraka) [reply w/wawa.md]: I have removed the spaces, the description of the genre and the [1] etc from the bio.

**Pull request merged**